lib/command.js

```diff
@@ -226,14 +232,6 @@ module.exports = function (proto) {
       return cb(e);
     }
     proc.stdin.once('error', cb);
-    
-    proc.on('error', function(err){
-      if (err.code === 'ENOENT') {
-        cb(new Error('Could not execute GraphicsMagick/ImageMagick: '+cmd+" this most likely means the gm/convert binaries can't be found"));
-      } else {
-        cb(err);
-      }
-    });
```

Comment:
Moved this block for clarity. It is only effective when `bufferOutput` is `true`. See discussion in #256.

Reply:
Makes sense. Just, to explain for the next reviewer:
- there's code to prevent the `callback` to be called more than once
- when `!buffer` case calls the callback immediately

So, when `!buffer` this error would be swallowed by the code that prevents the callback to be called more than once.